src/chrome/content/rules/Inc.com-falsemixed.xml

```diff
@@ -1,20 +0,0 @@
-<!--
-	For rules not causing MCB, see Inc.com.xml.
-
--->
-<ruleset name="Inc.com (mixed content)" platform="mixedcontent">
-
-	<target host="inc.com" />
-	<target host="conference.inc.com" />
-	<target host="mediakit.inc.com" />
```

Comment:
This target still has MCB. Can we keep it in this ruleset?

Reply:
I would like to avoid creating new `mixedcontent` rulesets since they are disabled by default anyway.